vagrant-spk

```diff
@@ -318,51 +323,29 @@ def call_vagrant_command(sandstorm_dir, *command_args):
         sys.exit(ANSI_RED + msg + ANSI_RESET)
 
 def ensure_working_vboxsf_in_base_box(sandstorm_dir):
-    # If the .sandstorm/Vagrantfile refers to debian/jessie64, and does not pin it to a particular
-    # version, and the user has version 8.2.2 available on their system, then we abort the
-    # "vagrant-spk up" process because it will not work.
+    # If the .sandstorm/Vagrantfile refers to debian/jessie64 or sandstorm/debian-jessie64
+    # then we abort the "vagrant-spk up" process because it will not work.
     def test_vagrantfile_refers_to_jessie64():
         with open(os.path.join(sandstorm_dir, 'Vagrantfile'), 'r') as fd:
             for line in fd:
                 if line.strip().split() == 'config.vm.box = "debian/jessie64"'.split():
                     return True
+                if line.strip().split() == 'config.vm.box = "sandstorm/debian-jessie64"'.split():
+                    return True
         return False
     vagrantfile_refers_to_jessie64 = test_vagrantfile_refers_to_jessie64()
 
-    def test_vagrantfile_pins_version():
-        with open(os.path.join(sandstorm_dir, 'Vagrantfile'), 'r') as fd:
-            for line in fd:
-                if line.strip().startswith('config.vm.box_version'):
-                    return True
-        return False
-    vagrantfile_pins_version = test_vagrantfile_pins_version()
-
-    def get_vagrant_dot_d_path():
-        if os.environ.get('VAGRANT_HOME'):
-            return os.environ.get('VAGRANT_HOME')
-        homedir = (os.environ.get('USERPROFILE') or os.environ.get('HOME'))
-        return os.path.join(homedir, '.vagrant.d')
-    user_has_version_822 = (os.path.exists(
-        os.path.join(get_vagrant_dot_d_path(), 'boxes/debian-VAGRANTSLASH-jessie64/8.2.2')))
-
-    if (vagrantfile_refers_to_jessie64 and
-            (not vagrantfile_pins_version) and
-            user_has_version_822):
+    if vagrantfile_refers_to_jessie64:
         msg = (
             """*** INCOMPATIBLE OPTIONS DETECTED ***
 
-You need to adjust .sandstorm/Vagrantfile due to recent incompatible changes
-made by a project we depend on. Apologies for this mess.
-
-Here's how to fix it:
-
-Open .sandstorm/Vagrantfile and find this line:
-
-  config.vm.box = "debian/jessie64"
-
-Change it to:
+This package was set to use a VM no longer compatible with vagrant-spk. You
+can either upgrade to the latest VM supported by vagrant-spk by running
+"vagrant-spk upgradevm" or manually update the package to use a compatible
+release of Debian Jessie.
```

Comment:
Vagrantfile says it uses stretch now, not jessie? Probably needs updating elsewhere too.

Reply:
This is intentional. You can _either_ use `upgradevm` to use the latest (currently: stretch) or we explain how to use the "correct" version of Jessie, if your app is really dependent on it being that and you want to make it work with Jessie.

Reply:
Definitely intending to add confirmation: https://github.com/sandstorm-io/vagrant-spk/issues/258

setupvm will already happily burn everything to the ground if you rerun it on an existing app.

Reply:
Aha, makes sense.